browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -219,27 +223,29 @@ RegisterPolymerTemplateModifications({
     }
 
     // Add 'Get Started' item
-    const peopleEl = getMenuElement(templateContent, '/people')
     const getStartedEl = createMenuElement(
       loadTimeData.getString('braveGetStartedTitle'),
       '/getStarted',
       'rocket',
       'getStarted'
     )
-    peopleEl.insertAdjacentElement('afterend', getStartedEl)
+    const peopleEl = getMenuElement(templateContent, '/people')
+    if (peopleEl) {
+      peopleEl.insertAdjacentElement('afterend', getStartedEl)
+    }
 
     // Move Appearance item
-    const appearanceBrowserEl = getMenuElement(templateContent, '/appearance')
-    getStartedEl.insertAdjacentElement('afterend', appearanceBrowserEl)
-
-    // Add Content item
     const contentEl = createMenuElement(
       loadTimeData.getString('contentSettingsContentSection'),
       '/braveContent',
       'window-content',
       'content',
     )
-    appearanceBrowserEl.insertAdjacentElement('afterend', contentEl)
+    const appearanceBrowserEl = getMenuElement(templateContent, '/appearance')
+    if (appearanceBrowserEl && contentEl) {
+      getStartedEl.insertAdjacentElement('afterend', appearanceBrowserEl)
+      appearanceBrowserEl.insertAdjacentElement('afterend', contentEl)
+    }
```

Comment:
do we want an 


```suggestion
    } else {
        console.error("Unabled to find appearance section - overrides are likely broken")
    }
```

similar for other entries. Or are there scenarios where they aren't added?

Reply:
It's a good point, but I think I'll keep it as-is for now as just about all of the insertions are missing an `else` clause like that. I'll create a follow-up issue to add those though!